packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx

```diff
@@ -16,7 +16,7 @@ import { GroupFooter } from './GroupFooter';
 
 import { List } from '../../List';
 import { IDragDropOptions } from './../../utilities/dragdrop/interfaces';
-import { css, getId } from '../../Utilities';
+import { css, getId, EventGroup } from '../../Utilities';
```

Comment:
Does this line needs to be consolidated with line 9?

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/pull/9810/commits/a14d23d193d1ad77fb6f6c24ab1cbea3b296964b